src/commands/load-buf-modules.ts

```diff
@@ -0,0 +1,59 @@
+import * as vscode from "vscode";
+
+import { Command, CommandType } from ".";
+import { BufFile } from "../context";
+import { unwrapError } from "../errors";
+import { execFile, log } from "../util";
+
+export const loadBufModules = new Command(
+  "buf.loadBufModules",
+  CommandType.COMMAND_INTERNAL,
+  (ctx, bufCtx) => {
```

Comment:
I think we need to have an explicit use-case for exposing the module name on the status bar as a tooltip before implementing this functionality. Currently, it seems like a nice-to-have and is a bit of a hidden feature that is not easy to discover, and this command would need a lot of documentation. I think we want to remove this in the current PR and scope this out in an issue as a follow-up.